llmfoundry/data/finetuning/tasks.py

```diff
@@ -920,10 +912,14 @@ def build_from_hf(
                 if not os.path.isdir(dataset_name):
                     # dataset_name is not a local dir path, download if needed.
                     local_dataset_dir = os.path.join(
-                        DOWNLOADED_FT_DATASETS_DIRPATH,
+                        tempfile.mkdtemp(),
                         dataset_name,
                     )
 
+                    log.debug(
+                        f'Downloading dataset {dataset_name} to {local_dataset_dir}.',
+                    )
```

Comment:
```suggestion
                    log.debug(f'Downloading dataset {dataset_name} to {local_dataset_dir}.')
```
coudl be one line

Reply:
unfortunately pre-commit is doing this 😢

Reply:
[221b8bd](https://github.com/mosaicml/llm-foundry/pull/1608/commits/221b8bd27a9156603a12e4e0160f9db5f87b7de4)